src/main/java/com/denizenscript/depenizen/bukkit/properties/auraskills/AuraSkillsPlayerExtensions.java

```diff
@@ -0,0 +1,118 @@
+package com.denizenscript.depenizen.bukkit.properties.auraskills;
+
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import dev.aurelium.auraskills.api.AuraSkillsApi;
+import dev.aurelium.auraskills.api.skill.Skill;
+import dev.aurelium.auraskills.api.skill.Skills;
+
+public class AuraSkillsPlayerExtensions {
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <PlayerTag.auraskills_levels>
+        // @returns MapTag
+        // @plugin Depenizen, AuraSkills
+        // @description
+        // Returns the skill levels of a player.
+        // @mechanism PlayerTag.auraskills_levels
+        // -->
```

Comment:
The meta here does not make much very obvious. Maybe include an example output? Especially on the mechs give an `@example`